### PR TITLE
time for a fresh beta

### PR DESCRIPTION
**Pull request by shiftkey**
Aiming to get this out tomorrow or Wednesday

- [x] enable repository indicators feature flag for beta
- [x] do some exploratory testing on Windows about loading testing new repository indicators

### CODE REVIEW

File: app/src/lib/feature-flag.ts [hunk 1]
```diff
@@ -39,5 +39,5 @@ export function enableNotificationOfBranchUpdates(): boolean {
 
 /** Should the repository list display info indicators? */
 export function enableRepoInfoIndicators(): boolean {
-  return enableDevelopmentFeatures()
+  return enableBetaFeatures()
```
Comment (nerdneha) [inline app/src/lib/feature-flag.ts]: Should this correspond with a 'new' for repo list info indicators?

Comment (shiftkey) [reply app/src/lib/feature-flag.ts]: It's currently in the list as an `Improved` entry:

```
[Improved] Repository list now displays uncommitted changes count and ahead/behind information - #2259
```

I don't have any strong feels on whether we want to call it out with `New` or save that for bigger-bang features. Thoughts @desktop/core @desktop/comrades?

Comment (niik) [reply app/src/lib/feature-flag.ts]: I'd definitely consider this to be a **new** entry

**Comment by shiftkey** [comment]
:apple:

**Comment by shiftkey** [comment]
Alright, I've just pushed a new commit to make this `1.3.0-beta1` - if we're happy with the words I'm ready to push this out and start getting feedback.

**Pull request merged**